### PR TITLE
Add tests for helper functions

### PR DESCRIPTION
**Pull request by deepansh946**
## Fixes
https://x-team-internal.atlassian.net/browse/CORE-2611

## How to test
- Run `npm run test` 
- It should pass without any errors.

## Screenshots

Screenshot 2024-10-17 at 4 26 50 PM

### CODE REVIEW

File: src/__tests__/signInAndOutHelper.test.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import { handleLogoutClick } from "../helpers/signInAndOutHelper";
```
Comment (nicksp) [inline src/__tests__/signInAndOutHelper.test.ts]: @deepansh946 I highly suggest we keep test files colocated with their respective modules for ease of navigation and to easier spot what's tested and what is not.

**Review by nicksp — CHANGES_REQUESTED** [review]

File: src/__tests__/signInAndOutHelper.test.ts (hunk 1 above)
Comment (nicksp) [reply src/__tests__/signInAndOutHelper.test.ts]: @deepansh946 I meant not moving the test folder but keeping the test files next to the modules they test so it's easy to spot which file has a corresponding test and which isn't without looking into nested folder.

**Review by nicksp — CHANGES_REQUESTED** [review]
Let's keep the test files colocated with their modules along each other for convenience.

**Review by nicksp — APPROVED** [review]

**Review by kengru — APPROVED** [review]

**Comment by nicksp** [comment]
@deepansh946 Please resolve merge conflicts.

**Comment by nicksp** [comment]
@deepansh946 It's ready to merge 🙌

**Pull request merged**